src/components/interactive-area/interactive-area.js

```diff
@@ -3,30 +3,64 @@ import {h, Component} from 'preact';
 import {withPlayer} from 'components/player';
 import style from '../../styles/style.scss';
 import {PlayerArea} from 'components/player-area';
+import {connect} from 'react-redux';
+
+/**
+ * mapping state to props
+ * @param {*} state - redux store state
+ * @returns {Object} - mapped state to this component
+ */
+const mapStateToProps = state => ({
+  guiClientRect: state.shell.guiClientRect,
+  topBarSize: state.topBar.topBarSize,
+  bottomBarSize: state.bottomBar.bottomBarSize
+});
 
 /**
  * InteractiveArea component
  *
  * @class InteractiveArea
  * @extends {Component}
  */
+@connect(mapStateToProps)
 @withPlayer
 class InteractiveArea extends Component {
   /**
-   * should component update handler
-   *
-   * @returns {boolean} shouldComponentUpdate
+   * this component should not render itself when player object changes.
+   * @param {Object} nextProps - next props of the component
+   * @returns {boolean} - should rerender state
+   * */
+  shouldComponentUpdate(nextProps: Object): boolean {
+    const {guiClientRect, topBarSize, bottomBarSize} = this.props;
+    const {nextPropsGuiStyles, nextPropsTopBarSize, nextPropsBottomBarSize} = nextProps;
+    return guiClientRect !== nextPropsGuiStyles || topBarSize !== nextPropsTopBarSize || bottomBarSize !== nextPropsBottomBarSize;
+  }
```

Comment:
you can just return `true` here. and actually, you can remove this method at all as it returns `true` by default.